NOTICE

```diff
@@ -37,13 +37,15 @@ PolyCrypt GmbH
     Jan Bormet <jan.bormet@bormet.info>
     Norbert Dzikowski <norbert@perun.network>
     Matthias Geihs <matthias@perun.network>
+    matthiasgeihs <62935430+matthiasgeihs@users.noreply.github.com>
```

Comment:
Why would you add my alternative GitHub email here? In particular, since I'm not involved in this PR, this seems wrong. I think the clean way would be to remove the respective author entry from the corresponding commit and force push.

Reply:
In general, please don't add any of my GitHub handles without my consent.

Reply:
Hm, did you try editing commit a865ac5fad117c665a7bc4ba504edd46de289fa0 and changing the author?

Anyhow, it seems like you resolved this somehow differently. Thank you!